virt/getting_started/virt-getting-started.adoc

```diff
@@ -104,5 +105,5 @@ Manage a VM:
 * xref:../../virt/storage/virt-storage-config-overview.adoc#virt-storage-config-overview[Configure storage options and automatic boot source updates].
 * xref:../../virt/monitoring/virt-monitoring-overview.adoc#virt-monitoring-overview[Learn about monitoring and health checks].
 * xref:../../virt/live_migration/virt-about-live-migration.adoc#virt-about-live-migration[Learn about live migration].
-* xref:../../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[Back up and restore VMs].
+// * xref:../../virt/backup_restore/virt-backup-restore-overview.adoc#virt-backup-restore-overview[Back up and restore VMs].
```

Comment:
Instead of commenting this link out, what do you think of doing this?

```suggestion
ifndef::openshift-rosa,openshift-dedicated[]
* xref:../../backup_and_restore/application_backup_and_restore/installing/installing-oadp-kubevirt.adoc#installing-oadp-kubevirt[Back up and restore VMs by using the {oadp-first}].
endif::openshift-rosa,openshift-dedicated[]
```